packages/picasso/src/InputLimitAdornment/test.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+import { render } from '@toptal/picasso/test-utils'
+
+import InputLimitAdornment, { Props } from './InputLimitAdornment'
+
+const renderAdornment = (props: Props) =>
+  render(<InputLimitAdornment {...props} />)
+
+describe('InputLimitAdornment', () => {
+  describe('multiline', () => {
+    it('uses singular when one characters is entered', async () => {
+      const { findByText } = renderAdornment({
+        counter: 'entered',
+        limit: 0,
+        multiline: true,
+        charsLength: 1,
+      })
+
+      const limitText = await findByText('1 character entered')
+
+      expect(limitText).toBeInTheDocument()
+    })
+
+    it('uses plural when two characters are entered', async () => {
```

Comment:
please, use this structure
```suggestion
  describe('when two characters are entered', () => {
    it('uses plural', async () => {
```